src/zenml/artifacts/data_drift_artifact.py

```diff
@@ -0,0 +1,21 @@
+#  Copyright (c) ZenML GmbH 2021. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+
+from zenml.artifacts.base_artifact import BaseArtifact
+
+
+class DataDriftArtifact(BaseArtifact):
```

Comment:
You're totally right. I had originally anticipated using this, but then decided it would have just added unnecessary complexity. When / if we have more monitoring integrations, we might consider standardising them via a class like you suggest, but I think I / we don't know enough what would go into that right now.

I have removed this class & references to it.